snap/hooks/pre-refresh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+
+INITIAL_PREVIOUS_VERSION_CONFIG="0.2.0"
+
+if [ -n "${DEBUG:-}" ]; then
```

Comment:
@adombeck question: Is `DEBUG` something you can enable in snap or is it for local testing?

Reply:
You can set the environment variable in the `snapcraft.yaml`  before building the snap:

```
environment:
  DEBUG: "true"
```